hybrid/utils.py

```diff
@@ -376,7 +376,7 @@ def select_random_subgraph(bqm, n):
         ['d', 'b']
 
     """
-    return random.sample(bqm.variables, n)
+    return random.sample(list(bqm.variables), n)
```

Comment:
huh, does this not work? We can probably fix that.

Reply:
Yeah, it's just that it's _also_ a sequence.

Reply:
Yup. But it's a _set_ first :laughing:

Reply:
It is deterministic. It just also has O(1) `v in variables` as well, so we mark it as being set-like.

Reply:
Not sure how to make that more obvious than having it inherit from `Sequence`?

Reply:
It might be obvious if you read the code. But if you write something like `random.sample(bqm.variables, n)`, and Python complains about using a set... Also if I'm aware something is a set, or something is a sequence, usually I don't assume they are both. Long story short, I can see how this will confuse some users.